### ISSUE TITLE
Disable "Sequence Filtering" globally

### ISSUE THREAD

**Issue by GithubMood** [database-rider/database-rider#252]
The "Sequence Filtering" feature automatically resolves the correct order of the INSERT/DELETE statements based on the foreign key metadata. This feature is enabled by default.

I noticed there is a huge performance overhead of using the feature.
Please see the benchmarks for the same test with the enabled and disabled feature.

Test with enabled "Sequence Filtering" feature:
![image](https://user-images.githubusercontent.com/23056623/102078668-f6273f00-3e13-11eb-89bf-d2b90cd98ab3.png)

Test with disabled "Sequence Filtering" feature:
![image](https://user-images.githubusercontent.com/23056623/102078819-3ab2da80-3e14-11eb-982e-f422a4a9cf19.png)

**Important remark: For disabling the "Sequence Filtering" feature, the order of the tables should be correct in the dataset!**
In our case, we maintain the correct order of the tables manually so we would like to disable the discussed feature.

Currently, there is only one option for disabling the feature using the DataSet attribute:
``` 
@DataSet(value = "dataset.xml", useSequenceFiltering = false)
```

It would be great to have the option for disabling the feature globally on the config level.
@rmpestano Please see PR with the proposed solution: #251

**Comment by rmpestano** [comment]
Thank you very much @GithubMood, it will be available in `1.20.0` probably to be released in next days. 

To use it right now you can use `1.19.1-SNAPSHOT` by enabling [snapshots repo](https://github.com/database-rider/database-rider#snapshots) in your build.

**Comment by GithubMood** [comment]
@rmpestano Thank you for your contribution and for maintaining such a great library!
